charts/substra-backend/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 <!-- towncrier release notes start -->
 
+## [26.4.0] - 2024-04-23
+
+### Added
+
+- Provides `.Values.kaniko.dockerConfigSecretName` to the executor (#862)
```

Comment:
Why do we need to update both the charts CHANGELOG and the towncrier `changes` folder?

Reply:
I mean:
1. Why do we update towncrier-related files if we still need to perform the old fashion manual updates? 
2. Why don't we leverage the use of towncrier in [substra-gha-workflows](https://github.com/Substra/substra-gha-workflows/blob/main/.github/workflows/helm.yml) so we don't bother with manual updates anymore? 😃

Reply:
I personally just make the update for the charts changelog by hand for now. Setting up towncrier just saved like 50 lines of `sed` on the automation script. 
That being said, I'm totally for leveraging the towncrier fragments in the gha workflow so that the changelog is generated automatically :) 
(I didn't even know where the GHA were defined before today 😅 )

Reply:
creating a card for that ⬆️